cmd/stat.go

```diff
@@ -453,10 +465,15 @@ func prettyPrintBucketMetadata(info BucketInfo) string {
 	placeHolder := ""
 	if info.Encryption.Algorithm != "" {
 		fmt.Fprintf(&b, "%2s%s", placeHolder, "Encryption: ")
-		fmt.Fprint(&b, console.Colorize("Key", "\n\tAlgorithm: "))
-		fmt.Fprint(&b, console.Colorize("Value", info.Encryption.Algorithm))
-		fmt.Fprint(&b, console.Colorize("Key", "\n\tKey ID: "))
-		fmt.Fprint(&b, console.Colorize("Value", info.Encryption.KeyID))
+		if info.Encryption.Algorithm == "aws:kms" {
+			fmt.Fprint(&b, console.Colorize("Key", "\n\tKey Type: "))
+			fmt.Fprint(&b, console.Colorize("Value", "SSE-KMS"))
+			fmt.Fprint(&b, console.Colorize("Key", "\n\tKey ID: "))
+			fmt.Fprint(&b, console.Colorize("Value", info.Encryption.KeyID))
+		} else {
+			fmt.Fprint(&b, console.Colorize("Key", "\n\tKey Type: "))
+			fmt.Fprint(&b, console.Colorize("Value", "SSE-S3"))
```

Comment:
Isn't there also `SSE-C`? Maybe this instead?

```suggestion
			fmt.Fprint(&b, console.Colorize("Value", strings.ToUpper(info.Encryption.Algorithm)))
```

Reply:
I believe SSE-C can't be set on a bucket, so only KMS and S3 are options. S3 algorithm is AES256, so have it print as SSE-S3 when printing as a string for simplicity.

Reply:
Keep it generic use the value from `info.Encryption.Algorithm`